src/framework/global/io/fileinfo.cpp

```diff
@@ -145,37 +145,25 @@ QDateTime FileInfo::lastModified() const
     return fileSystem()->lastModified(m_filePath);
 }
 
-#if defined(Q_OS_WIN)
 bool FileInfo::isRelative() const
 {
-    return m_filePath.isEmpty()
-           || (m_filePath.at(0) != '/'
-               && !(m_filePath.length() >= 2 && m_filePath.at(1) == ':'));
+    return !isAbsolute();
 }
 
 bool FileInfo::isAbsolute() const
 {
+#ifdef Q_OS_WIN
     return (m_filePath.length() >= 3
             && m_filePath.at(0).isLetter()
             && m_filePath.at(1) == ':'
             && m_filePath.at(2) == '/')
-           || (m_filePath.length() >= 2
-               && m_filePath.at(0) == '/'
-               && m_filePath.at(1) == '/');
-}
-
+           || (!m_filePath.isEmpty()
+               && (m_filePath.at(0) == '/'
```

Comment:
A path starting with \ is relative in windows - but relative to the current *drive*, not the current directory.
Whereas \\?\c:\xyz is absolute, as is \\hostname\share\